packages/bson-bench/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "bson-bench",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./lib/index.js",
+  "scripts": {
+    "build": "npx tsc",
+    "test": "npm run build && npx mocha test/unit/*.test.ts",
+    "check:eslint": "eslint -v && eslint --max-warnings=0 --ext '.js,.ts' src test",
+    "fix:eslint": "npm run check:eslint -- --fix"
+  },
+  "engines": {
+    "node": ">=16.20.1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@tsconfig/node16": "^16.1.1",
+    "@types/chai": "^4.3.7",
+    "@types/mocha": "^10.0.2",
+    "@types/node": "^16.18.58",
+    "@typescript-eslint/eslint-plugin": "^6.8.0",
+    "@typescript-eslint/parser": "^6.8.0",
+    "chai": "^4.3.10",
+    "eslint": "^8.51.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-import": "^2.28.1",
+    "eslint-plugin-prettier": "^5.0.1",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
+    "eslint-plugin-tsdoc": "^0.2.17",
+    "eslint-plugin-unused-imports": "^3.0.0",
+    "mocha": "^10.2.0",
+    "prettier": "^3.0.3",
+    "ts-node": "^10.9.1",
+    "typescript": "^5.2.2",
+    "typescript-cached-transpile": "^0.0.6"
+  },
+  "dependencies": {
+    "bson": "^6.1.0",
+    "npm": "^9.9.0"
```

Comment:
I see the `pathToNpm` logic, can we not trust that the environment setup has the correct npm bin?

Reply:
My logic here was to make sure we have on npm 9 available so that the hardcoded syntax here will work regardless of where it has to run, but we can remove this dependency if we don't think this necessary.

Reply:
I'm in agreement with Neal here - I think we can let the environment dictate the version and force it by pinning the npm version in the `engines` field instead.